lighthouse-core/test/fixtures/config-plugins/lighthouse-plugin-no-category/plugin-no-category.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/** @type {LH.Config.Plugin} */
+module.exports = {
+  audits: [
+    {path: 'redirects'},
```

Comment:
this wouldn't be valid right? or would this try to add a copy of our internal redirects audit?

Reply:
> this wouldn't be valid right? or would this try to add a copy of our internal redirects audit?

It does become a copy of our internal redirects audit. LIke the SEO category reuses audits from other categories (`viewport`, `a11y/document-title`), it seemed reasonable to let plugins reuse core audits (especially if their plan is to drop all our normal categories and just use their category).

Is the `path` part of that deceiving when it's not actually a path? My justification has been that it's kind of like `require()`, if it's an internal node module you don't need a path (like if it's a core audit, you don't need a name), but maybe that's just going to cause confusion.

Reply:
Yeah no that makes sense, and since it's extension it'll be de-duped. It's a bit problematic though that a plugin could override the settings of a core audit :/ Maybe we should make a copy and automatically give it a new namespaced `auditRef`?

Reply:
> Bummer they can't use a copy with different settings then

I was half thinking we could easily open it up to that later on, but yeah, I guess that won't work with our current approach to options/audit ids. We should think about that for the future :)